libsolidity/ast/ASTUtils.h

```diff
@@ -48,4 +48,7 @@ Type const* type(Expression const& _expression);
 /// (this can happen for variables with non-explicit types before their types are resolved)
 Type const* type(VariableDeclaration const& _variable);
 
+/// @returns the size occupied by the variables at given _location (either Storage or Transient Storage).
```

Comment:
One more thing: we should clarify that this is the whole hierarchy, not just the contract. And that it's the number of slots, not size in bytes:

```suggestion
/// @returns The number of slots occupied by all state variables in contract's inheritance hierarchy,
/// located in @a _location (either storage or transient storage).
```